Enable CI builds

rtyley opened the pull request:
This is preparation for getting Riff Raff to deploy the interactive. These first steps:
- Remove hard-coding of the deployment path in the artifact (RiffRaff depeneds on artifacts being identical, no matter what stage they're deployed to)
- Give a single `package` task - https://github.com/jonnyzzz/TeamCity.Node failed to parse multiline tasks anyway.

The new CI task is at: https://teamcity-aws.gutools.co.uk/viewType.html?buildTypeId=memsub_membership_InteractiveBrexitCompanion

Once this is merged, I'll enable feedback from teamCity to the GitHub PRs, so we can see if PRs pass the build process.

cc @annebyrne @joelochlann @SiAdcock

rtyley committed:
Change the deployable files so they don't contain their deployed path

For RiffRaff we want to have a deployable artifact that will work for
*any* environment, so it doesn't hardcode the destination into it's own
deployable files, like this:

```
embed.html
6:    <link rel="stylesheet" type="text/css" href="https://interactive.guim.co.uk/testing/2016/05/brexit-companion/1464873660958/embed.css" />
10:    <script src="https://interactive.guim.co.uk/testing/2016/05/brexit-companion/1464873660958/embed.js"></script>
12:        init(document.querySelector('.js-interactive'), {'assetPath': 'https://interactive.guim.co.uk/testing/2016/05/brexit-companion/1464873660958'});
```

rtyley committed:
Create new Grunt task to allow CI to package deployable files

This is based off the existing 'deploy' task, but for RiffRaff, we want the
CI build to just create a packaged artifact, not deploy the code.

joelochlann commented on Gruntfile.js:
```diff
@@ -216,7 +216,7 @@ module.exports = function(grunt) {
             timestamp: Date.now(),
             jspmFlags: '-m',
-            assetPath: '<%= visuals.s3.domain %><%= visuals.s3.path %>/<%= visuals.timestamp %>'
+            assetPath: '<%= visuals.timestamp %>'
```
As discussed, because of lines 173 and 187 where the assets and the `embed.html` get deployed to different directories, you'd need to go one directory up to successfully use a relative path here

rtyley committed:
Cope with the fact that embed.html is put one folder further down

`embed.html` is the one place where the relative asset path is used
at the moment.

https://github.com/guardian/interactive-brexit-companion/pull/34/files/c26f1bab9ebe2a6b673dae2f5ca64d614048b524#r65693223

joelochlann commented:
👍

rtyley commented:
Looks OK in test:

https://interactive.guim.co.uk/testing/2016/05/brexit-companion/embed/embed.html?id=impact_economy
![image](https://cloud.githubusercontent.com/assets/52038/15779160/9e216000-2992-11e6-8cb0-0d95a6d73458.png)
